### DISCUSSION TITLE
About Random testing for unreliable case

### DISCUSSION THREAD

**Discussion by overflow218** [ANLAB-KAIST/KENSv3#82] [Project Q&A 2. Transport Layer (TCP)] [answered]
Dear TA
I saw another student's post about random testing. 
I want to check if my code is running well for various random situations. 
Is it enough to just change random seed in transfer.cpp to whatever I want random value?
 unsigned int seed = atoi(env["RANDOM_SEED"].c_str());
or is there other good way to check it? 
Thank you!

**Comment by Pusnow** [comment, marked as answer]
You can use environment variable to set `RANDOM_SEED`.

```bash
$ RANDOM_SEED=1234 ./app/kens/test-kens-transfer-unreliable
```

or

```bash
$ export RANDOM_SEED=1234
$ ./app/kens/test-kens-transfer-unreliable
```



> Is it enough to just change random seed in transfer.cpp to whatever I want random value?

Yes.

**Reply by overflow218** [reply]
Thank you!
